packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -42,14 +42,41 @@ keywords: [menu, dropdown, action menu]
 ## Example
 
 ```tsx example
+/**
+ * Temporary, for testing submenu positioning in docs
+ * TODO: Remove before merging into main.
```

Comment:
Since this bug was most easily seen in the docs ([PR](https://github.com/adobe/react-spectrum/pull/4763)), I've added an example temporarily to make testing this easier.